brion/blueConfig.cpp

```diff
@@ -65,6 +68,12 @@ inline std::string lexical_cast( const brion::BlueConfigSection& b )
 }
 }
 
+namespace
+{
+    const char* const SPIKE_FILE = "/out.dat";
```

Comment:
Public header sounds good. People and tests can use it directly as well then w/o the convenience functions if they want to.